### PR TITLE
created dropdown allowing users to switch session.

### PR DESCRIPTION
**Pull request by Cazaril**

### CODE REVIEW

File: public/resources/core/js/controllers/user.controller.js [hunk 1]
```diff
@@ -59,11 +61,47 @@
...
+	}
+
+	function showOrgList(orgId) {
+	    return vm.currentUser.id != orgId;
+	}
+
+	function switchSession(orgId) {
+	    var currUser = User.loggedUser.organizations.find(x => x.id == orgId);
+	    vm.currentUser.name = currUser.name;
+	    vm.currentUser.email = currUser.email ? currUser.email : "email@example.com";
```
Comment (fdelavega) [inline public/resources/core/js/controllers/user.controller.js]: Why are you including a default email?

File: views/base.jade [hunk 2]
```diff
@@ -60,7 +60,19 @@ html(ng-app="app")
...
             email: '#{ user.email }',
-            bearerToken: '#{ user.accessToken }'
+            bearerToken: '#{ user.accessToken }',
+            organizations: '#{ '{ "organizations":' + JSON.stringify(user.organizations) + '}' }',
+            currentUser: {
+              id: '#{ user.id }',
+              name: '#{ user.displayName }',
+              email: '#{ user.email }'
+            },
+            logInfo: {
```
Comment (fdelavega) [inline views/base.jade]: This object is redundant, you can directly use the information included in LOGGED_USER

File: public/resources/core/js/controllers/user.controller.js [hunk 3]
```diff
@@ -59,11 +61,47 @@
...
+	vm.showOrgList = showOrgList;
+	vm.showOrgs = false;
+	vm.hasAdminRole = hasAdminRole;
+
+	function hasAdminRole() {
+	    var org = User.loggedUser.organizations.find(x => x.id == vm.currentUser.id);
+	    return !loggedAsIndividual() || org.roles.findIndex(x => x.name == "Admin") > -1;
+	}
+
+	function loggedAsIndividual() {
```
Comment (fdelavega) [inline public/resources/core/js/controllers/user.controller.js]: This method has a confuse name, is called loggedAsIndividual, but it returns false when your are logged as an individual

File: public/resources/core/js/controllers/user.controller.js (hunk 1 above)
Comment (Cazaril) [inline public/resources/core/js/controllers/user.controller.js]: I thought it was a required field for organizations and placed that default example email as a reminder.

**Pull request merged**